.github/workflows/canary.yml

```diff
@@ -0,0 +1,92 @@
+name: Canary
+on:
+  pull_request:
+    branches-ignore:
+      - "master"
+env:
+  GITHUB_TOKEN: ${{ secrets.ADMIN_TOKEN }}
+  GH_TOKEN: ${{ secrets.ADMIN_TOKEN }}
+  NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    name: Build
+    if: "!contains(github.event.head_commit.message, 'ci skip') && !contains(github.event.head_commit.message, 'skip ci')"
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.ADMIN_TOKEN }}
+
+      - name: Prepare repository
+        run: git fetch --unshallow --tags
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'yarn'
+
+      - name: Install dependencies
+        run: |
+          yarn -v
+          yarn install --immutable
+
+      # - name: Apply semver updates (if any)
```

Comment:
Let's also ensure we drop this before merge, we should not keep commented out blocks (at least withoput TODOs)